tap_twilio/sync.py

```diff
@@ -101,7 +104,7 @@ def process_records(catalog,  # pylint: disable=too-many-branches
                     bookmark_dttm = transform_datetime(transformed_record[bookmark_field])
                     # Keep only records whose bookmark is after the last_datetime
                     if bookmark_dttm:
-                        if bookmark_dttm >= last_dttm:
+                        if bookmark_dttm > last_dttm:
```

Comment:
If no records are added/updated tap code should atleast retrieve the single record (with the same bookmark value). If yes, then why are we doing this change?

Reply:
I don't think that is the case. there can be multiple records whose replication_key value would be same and if u have `>=` all those records get replicated in sub-sequent extractions.

Reply:
okay. Please test this change in a bookmarking integration test.

Reply:
The bookmarking logic should remain as `>=`. If any records with a value of `last_dttm` were added after the last sync, they would be missed in the subsequent one. Its an accepted practice that we will get duplicate records on the bookmarked value.

Reply:
Reverted it